src/Routes/RosDetailsPage/RosDetailsPage.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Check how the inventory detail is done in inv application https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/src/routes/InventoryDetail.js or in edge https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/DeviceDetail/DeviceDetail.js you don't have to write the top part as this is the same for all system details, those `InventoryDetailHead, AppInfo, DetailWrapper,` components will do all the heavy work for you.

Reply:
@karelhala Applying references from both edge and inventory as suggested (still WIP), but, I wanted to know if we would need to define something similar to [groupDetailsReducer](https://github.com/RedHatInsights/edge-frontend/blob/master/src/Routes/DeviceDetail/DeviceDetail.js#L29) for creating entities and using those here?

Reply:
Not really, that reducer just takes the name of a group to be used in the breadcrumbs.